### ISSUE TITLE
[HOLD for payment 2025-01-30] [$250] [New Feature] Add Guide booking link to #admins header

### ISSUE THREAD

**Issue by muttmuure** [Expensify/App#53764]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v9.0.73-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:**
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @muttmuure 
**Slack conversation** (hyperlinked to channel name): https://expensify.slack.com/archives/C07HPDRELLD/p1732636200993259

## Action Performed:
Open #admins room for Workspace

## Expected Result:

See Guide/Account Manager booking link in header

## Actual Result:

N/A (this is a new feature)

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

![image](https://github.com/user-attachments/assets/e3cd8b85-e00c-4b61-90e5-f2a82b8216ce)

Just an idea

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021869030905695211861</li>
        <li>Upwork Job ID: 1869030905695211861</li>
        <li>Last Price Increase: 2024-12-17</li>
<li>Automatic offers: </li>
<ul>
<li>nkdengineer | Contributor | 105383951</li>
</ul></li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @anmurali</details>

**Comment by ChavdaSachin** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Add Guide booking link to #admins header

### What is the root cause of that problem?
New Feature

### What changes do you think we should make in order to solve the problem?
Conditionally render `Book A Demo` button from HeaderView.tsx .

add following snippet [here](https://github.com/Expensify/App/blob/dc07fc67e8064b5941f7f8467d9bd5403c81f817/src/pages/home/HeaderView.tsx#L84)

```js
    const shouldShowDemoButton = ReportUtils.isAdminRoom(report);
    const bookDemo = () => {
        Linking.openURL(CONST.BOOK_DEMO_URL);
    }
```
And conditionally render button here. (Pass success prop based on design team's suggestion.
```js
    {shouldShowDemoButton &&
        <Button
        success
        onPress={bookDemo}
        icon={}//add  whatever icon design team suggests
        text={translate('report.common.bookDemo')}
        style={[shouldUseNarrowLayout && styles.flex1]}
        />
    }
```
https://github.com/Expensify/App/blob/dc07fc67e8064b5941f7f8467d9bd5403c81f817/src/pages/home/HeaderView.tsx#L295-L296

And add translations to en.ts and es.ts file and add url to consts.ts file(currently there is no direct url available for booking a general demo with team, url for travel demo is avilable).

And do some small UI adjustments as design team demands.

### What specific scenarios should we cover in automated tests to prevent reintroducing this issue in the future?

NA

### What alternative solutions did you explore? (Optional)

**Reminder:** Please use plain English, be brief and avoid jargon. Feel free to use images, charts or pseudo-code if necessary. Do not post large multi-line diffs or write walls of text. Do not create PRs unless you have been hired for this job.

**Comment by shawnborton** [comment]
We could just use a green button much like we do for the "Start a free trial" button on the Concierge view? 
![CleanShot 2024-12-09 at 15 16 16@2x](https://github.com/user-attachments/assets/863493a9-fbf9-4417-bcd0-19ca298220e6)

**Comment by dannymcclain** [comment]
Will this live along side the [early adoption discount](https://github.com/Expensify/Expensify/issues/440617)?

**Comment by joekaufmanexpensify** [comment]
Seems like I was assigned as a second BZ assignee on this issue. I don't think it needs two though, so going to unassign for now. LMK if there is anything I can help with though!

**Comment by dubielzyk-expensify** [comment]
Great Qs from the design team. Agree that the treatment Shawn is using is the right one and as long as early adoption discount isn't in the #admin room, then that could work.

![image](https://github.com/user-attachments/assets/d8001330-07ed-4295-887c-2d8e9e8e59b6)

**Comment by nkdengineer** [comment]
@dubielzyk-expensify Is the icon is an existing icon in the App?

**Comment by dubielzyk-expensify** [comment]
Yep. [Here it is](https://github.com/Expensify/App/blob/main/assets/images/calendar-solid.svg)

**Comment by shawnborton** [comment]
Mock looks great!

**Comment by shawnborton** [comment]
@muttmuure at what point will we not show this button anymore? Once the user has taken a demo? Or once the user converts to a paid policy? Something else?

**Comment by muttmuure** [comment]
I think once they convert to a paid policy makes the most sense.

**Comment by muttmuure** [comment]
It's common for customers to take multiple demos

**Comment by dubielzyk-expensify** [comment]
Mobile mock:
![image](https://github.com/user-attachments/assets/76a8cf30-35b6-46e3-8f4f-74398bc30eb1)

**Comment by shawnborton** [comment]
Looks great!

**Comment by muttmuure** [comment]
Ready for the EXTERNAL label?

**Comment by dubielzyk-expensify** [comment]
Let's do it

**Comment by eh2077** [comment]
I think we can go with @nkdengineer 's [proposal](https://github.com/Expensify/App/issues/53764#issuecomment-2529003390) because their proposal is more thoughtful than the other one. 

> [!NOTE]  
> https://github.com/Expensify/App/issues/53764#issuecomment-2531736033
> > @muttmuure at what point will we not show this button anymore? Once the user has taken a demo? Or once the user converts to a paid policy? Something else?
>  > > I think once they convert to a paid policy makes the most sense.


🎀👀🎀 C+ reviewed

**Comment by grgia** [comment]
Thanks for your proposal @nkdengineer, all yours!

**Comment by grgia** [comment]
PR in final review

**Comment by Beamanator** [comment]
@dubielzyk-expensify looks like the main PR has a small bug on very small screens - https://github.com/Expensify/App/issues/55522 - can you let @nkdengineer know what the UI should look like on super small screens?

**Comment by eh2077** [comment]
# BugZero Checklist:

- [ ] **[Contributor]** Classify the bug:

<details>
<summary>Bug classification</summary>


Source of bug:
  - [ ] 1a. Result of the original design (eg. a case wasn't considered)
  - [ ] 1b. Mistake during implementation
  - [ ] 1c. Backend bug
  - [ ] 1z. Other:

Where bug was reported:
  - [ ] 2a. Reported on production (eg. bug slipped through the normal regression and PR testing process on staging)
  - [ ] 2b. Reported on staging (eg. found during regression or PR testing)
  - [ ] 2d. Reported on a PR
  - [ ] 2z. Other:

Who reported the bug:
  - [ ] 3a. Expensify user
  - [ ] 3b. Expensify employee
  - [ ] 3c. Contributor
  - [ ] 3d. QA
  - [ ] 3z. Other:

</details>

- [ ] **[Contributor]** The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake.

    Link to comment:

- [ ] **[Contributor]** If the regression was CRITICAL (e.g. interrupts a core flow) A discussion in [#expensify-open-source](https://app.slack.com/client/E047TPA624F/C01GTK53T8Q) has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner.

    Link to discussion:

- [ ] **[Contributor]** If it was decided to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps using the template below to ensure the same bug will not reach production again.

<details>
<summary>Regression Test Proposal Template</summary>


- [ ] **[BugZero Assignee]** Create a GH issue for creating/updating the regression test once above steps have been agreed upon.

    Link to issue:

## Regression Test Proposal
### Precondition:


-

### Test:


1.

Do we agree 👍 or 👎


</details>

**Comment by grgia** [comment]
@anmurali this one is ready for payment / regression tests

**Comment by eh2077** [comment]
@anmurali Friendly bump!

**Comment by JmillsExpensify** [comment]
$250 approved for @eh2077
